db/schema.rb

```diff
@@ -595,4 +595,4 @@
   end
 
   add_foreign_key "course_stats", "courses"
-end
+end
```

Comment:
Ideally, this should not be part of the changeset.

Reply:
I'm not sure why my db.schema gets updated, cause I didn't perform any changes in it essentially. It seemed to have gotten updated when I launched the server.

Reply:
In this case, it's just whether there's a final newline at the end of the file, which is something that usually happens automatically (or not) and varies by operating system. This change doesn't actually matter, but I'd rather have a clean PR.

Reply:
Sure I completely understand, I did switch my operating system cause I running into errors and was having trouble setting up the dev environment initially on M1 but I'll make sure that it doesn't happen again.